tests/lib/prepare.sh

```diff
@@ -637,10 +637,14 @@ prepare_ubuntu_core() {
     fi
 
     echo "Ensure the core snap is cached"
+    # Cache snaps
     if is_core18_system; then
-        if ! snap list core; then
-            cache_snaps core
+        if snap list core; then
```

Comment:
This will display information about core. Perhaps you want to redirect the output to /dev/null?

Reply:
Done